tools/autoformat

```diff
@@ -2,6 +2,23 @@
 
 set -euo pipefail
 
+NO_PARALLEL=false
```

Comment:
I'd rather avoid the double negation here.

`USE_PARALLEL=true`

Reply:
ok. `NO_PARALLEL` has been a leftover from the previous version that had a `-no-parallel` switch